conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -6,6 +6,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
+  - rapidsai/label/cuda10.0::cupy>=6*
   - rmm=0.10.*
```

Comment:
Can you bump this to 0.11 in the PR as well?

Reply:
done